plugins/jobs/girder_plugin_jobs/models/job.py

```diff
@@ -24,7 +24,9 @@
 from girder import events
 from girder.constants import AccessType, SortDir
 from girder.models.model_base import AccessControlledModel, ValidationException
-from girder.plugins.jobs.constants import JobStatus, JOB_HANDLER_LOCAL
+from girder.utility.model_importer import girder_model
+
+from ..constants import JobStatus, JOB_HANDLER_LOCAL
```

Comment:
`girder.plugins` was removed so we use a relative import, or alternatively an absolute import from the actual package name `girder_plugin_jobs.constants`.